buildSrc/src/main/kotlin/io.opentelemetry.instrumentation.javaagent-instrumentation.gradle.kts

```diff
@@ -0,0 +1,10 @@
+plugins {
+  id("io.opentelemetry.instrumentation.javaagent-testing")
+  id("io.opentelemetry.instrumentation.muzzle-check")
+}
+
+dependencies {
+  add("muzzleBootstrap", "io.opentelemetry.javaagent:opentelemetry-javaagent-bootstrap")
```

Comment:
As @mateuszrzeszutek said, `instrumentationMuzzle` seemed to be equivalent to `runtimeClasspath`, which is equivalent to just having a normal dependency like this one